### PR TITLE
ref(app): change attestation certificate format from b64 to hex string

### PR DESCRIPTION
**Pull request by ubordignon**

### CODE REVIEW

File: app/src/main/java/proofcastlabs/tee/security/Strongbox.java [hunk 1]
```diff
@@ -383,12 +385,8 @@ public String getCertificateAttestation() {
             CBORFactory cborFactory = new CBORFactory();
             ObjectMapper mapper = new ObjectMapper(cborFactory);
             byte[] data = mapper.writeValueAsBytes(attestationCertificate);
-            ByteArrayOutputStream ba = new ByteArrayOutputStream();
-            ba.write(data);
 
-            return Base64
-                    .encodeToString(ba.toByteArray(), Base64.DEFAULT)
-                    .replaceAll("\n", "");
+            return new String(Hex.encodeHex(data));
```
Comment (gitmp01) [inline app/src/main/java/proofcastlabs/tee/security/Strongbox.java]: Shall we prefix it with `0x`?

**Review by gitmp01 — APPROVED** [review]
Comment + LGTM

File: app/src/main/java/proofcastlabs/tee/security/Strongbox.java (hunk 1 above)
Comment (ubordignon) [reply app/src/main/java/proofcastlabs/tee/security/Strongbox.java]: I don't have a preference, I checked other usages of `encodeHex` in the repo and none of them seem to be adding `0x` explicitly. Up to you!

Comment (gitmp01) [reply app/src/main/java/proofcastlabs/tee/security/Strongbox.java]: I'd add it, yeah

Comment (ubordignon) [reply app/src/main/java/proofcastlabs/tee/security/Strongbox.java]: Done!

**Pull request merged**